### PR TITLE
[Feedback Fix] Update site statuses

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # 176695364 #176695371

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Desktop Screenshots
Screenshot 2021-02-02 at 13 39 59

image

Screenshot 2021-02-02 at 13 39 09

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/Hero/ProfileChart/DesktopChart/index.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const useStyles = makeStyles(({ typography }) => ({
 
 function DesktopChart({ promisesByStatus, ...props }) {
   const classes = useStyles(props);
+  console.log(promisesByStatus);
```
Comment (kilemensi) [inline src/components/Hero/ProfileChart/DesktopChart/index.js:31]:
```suggestion
  console.log(promisesByStatus);
```
`yarn lint` anyone?

File: src/components/Hero/ProfileChart/RectChart/index.js [hunk 2]
```diff
@@ -13,12 +13,12 @@ const useStyles = makeStyles(() => ({
...
 function ReactChart({
   totalPromises,
-  completed,
+  complete,
   inconclusive,
   inProgress,
   unstarted,
   stalled,
-  delayed,
+  behind,
```
Comment (kilemensi) [inline src/components/Hero/ProfileChart/RectChart/index.js:21]:
```suggestion
  behindSchedule,
```
`behind` what?

**Review by kilemensi — APPROVED** [review]
🚀 @nerdlet 

________________________________

Mobile looks "cut off" though
Screenshot from 2021-02-02 14-59-51

File: src/components/Hero/ProfileChart/DesktopChart/index.js (hunk 1 above)
Comment (kilemensi) [reply src/components/Hero/ProfileChart/DesktopChart/index.js:31]: Ping @nerdlet

File: src/components/Hero/ProfileChart/RectChart/index.js [hunk 3]
```diff
@@ -109,20 +109,20 @@ function ReactChart({
 }
 
 ReactChart.propTypes = {
-  delayed: PropTypes.number,
+  behind: PropTypes.number,
```
Comment (kilemensi) [inline src/components/Hero/ProfileChart/RectChart/index.js:112]: Shouldn't these also change to `behindSchedule`?

Comment (nerdlet) [reply src/components/Hero/ProfileChart/RectChart/index.js]: @kilemensi Yap changed them

File: src/theme/index.js [hunk 4]
```diff
@@ -25,11 +25,11 @@ const theme = createTheme({
     highlight: { main: "#005DFD", light: "#FFB322", faded: "#90DAFF" }, // blue, yellow, light blue
     text: { primary: "#202020", secondary: "#ffffff" },
     chart: {
-      completed: "#005DFD",
+      complete: "#005DFD",
       inprogress: "#90DAFF",
       inconclusive: "#909090",
       unstarted: "#EBEBEB",
-      delayed: "#FFB322",
+      behindschedule: "#FFB322",
```
Comment (kilemensi) [inline src/theme/index.js:32]:
```suggestion
      behindSchedule: "#FFB322",
```

**Review by kilemensi — APPROVED** [review]
:rocket:

**Pull request merged**